project/Extra.scala

```diff
@@ -45,8 +43,20 @@ object Extra {
           "scm:git:git@github.com:portable-scala/sbt-crossproject.git"
       )
     ),
-    bintrayRepository := "sbt-plugins",
-    bintrayOrganization := Some("portable-scala")
+    // Publish to Bintray, without the sbt-bintray plugin
+    publishMavenStyle := false,
+    publishTo := {
```

Comment:
LOL this is too easy.

Reply:
Yep, it seems like a pain of suffering credential warnings from bintray-sbt was largely self-inflicted.